### ISSUE TITLE
Apple TV Captions hidden when extension is active

### ISSUE THREAD

**Issue by bdc0der** [FrostCo/AdvancedProfanityFilter#547]
#### :bug: Description

I have noticed (for about the past few days) that the extension messes up the Apple TV website. Specifically, if the extension is active, the captions won't appear. 

#### :twisted_rightwards_arrows: Steps To Reproduce

1. Go to tv.apple.com
2. Click on any video (I chose Palm Royale as an example)
3. Make sure captions are enabled and the extension is active
4. See the captions hidden

#### :heavy_check_mark: Expected behavior

I expected the captions to be present, but they were hidden, which made it impossible to filter the audio. 

#### :camera: Screenshots

<img width="1440" alt="Screenshot 2024-03-23 at 6 18 14 PM" src="https://github.com/FrostCo/AdvancedProfanityFilter/assets/119639394/a39a39e8-dc78-43ff-b3fd-7d5c3b09e1e7">

<img width="1440" alt="Screenshot 2024-03-23 at 6 21 08 PM" src="https://github.com/FrostCo/AdvancedProfanityFilter/assets/119639394/99b5444e-dc31-4816-84a4-2ad4515e915f">#### :computer: Details

- **Device:** MacBook Pro & Windows
- **OS:** MacOS 13.6.3 & Windows 10
- **Browser:** Brave & Chrome
- **Browser Version:** 
Brave: Version 1.64.109 Chromium: 123.0.6312.58 (Official Build) (arm64) 
- Chrome: 123.0.6312.59 (Official Build) (64-Bit)
- **Affected site(s):** tv.apple.com

#### :pencil: Additional context

I've double-checked to make sure I have the captions enabled, and I've tried using a brand-new extension profile on a brand-new browser profile. (I now use Brave for testing stuff, and Chrome for personal use)

**Comment by richardfrost** [comment]
Thanks for reporting the issue. I think the problem is actually two-fold. For Apple TV, APF has actually been hiding the captions for quite a while now, but it is supposed to show its own version of them on the page. Due to other behaviors that Apple TV had, it was the best option I had to make it work with the page. It seems that they have changed their page layout though, and it has broken it in such a way as to hide all the captions, but not detect that there are actually any, so it isn't showing any. Sorry for the long explanation.

I will try to build out an updated config in the next day or two, and I'll post it here so you can try it out.

**Comment by bdc0der** [comment]
No problem. 
Sounds good, I eagerly await your results.

**Comment by richardfrost** [comment]
I can't test this right now,  but if you have some time @bdc0der feel free to give it a try:

```json
{
  "tv.apple.com": [
    {
      "apfCaptions": true,
      "apfCaptionsSelector": "div.video-container",
      "captionControlStyle": true,
      "displaySelector": "div.video-player div.video-container > div > div",
      "mode": "watcher",
      "subtitleSelector": "div.video-player div.video-container > div > div > div > div",
      "videoSelector": "div.video-player div.video-container video"
    }
  ]
}
```

To use this config, paste it into the Custom Sites area (Options page > Audio tab) and click the blue "SAVE" button underneath it. Then, make sure to refresh the tab for the video site (if you have it open still) and it will load this new configuration.

**Comment by bdc0der** [comment]
This configuration works perfectly on both the devices and browsers I used. Thanks for the quick response!

**Comment by richardfrost** [comment]
You're welcome! Thanks for confirming that it's working. I'll get this added into the next version of the extension. Once you get the update, I'd recommend removing the custom config provided here, just in case it changes again in the future. Until then though, feel free to use this without any problems.

**Comment by bdc0der** [comment]
Sounds good. I'll keep this open for a few more days in case of any problems. Otherwise, thanks for the help!

**Comment by richardfrost** [comment]
I have included this latest config in [Version 3.4.1](https://github.com/FrostCo/AdvancedProfanityFilter/releases/tag/3.4.1). Feel free to respond here if you have any more feedback (good or bod). I'll leave this open for a few more days before closing it. Thanks @bdc0der for your help in reporting the issue and testing the fix.
